components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -23,6 +25,24 @@ export const formatConverted = (converted: string, currency: string = 'USD'): st
   return `${converted} ${currency}`
 }
 
+export const handleContributionAmount = (amount: string) => {
+  let result = '0.0'
+  const amountSplit = amount.split('.')
+  if (amountSplit && amountSplit[0].length > 18) {
```

Comment:
`>= 19` would be easier to read as probi are always 19 characters

Reply:
`>= 19` and `>18` both tell you that we are looking for number 19.  Personally I like to avoid `>=` as it's really easy to miss when reading through stuff

Reply:
Easier to read if the magic number is replaced